apt-cacher/Dockerfile

```diff
@@ -0,0 +1,16 @@
+#
+# Build: docker build -t apt-cacher .
+# Run: docker run -d -p 3142:3142 --name apt-cacher-run apt-cacher
+#
+# and then you can run containers with:
+#   docker run -t -i --rm -e http_proxy http://dockerhost:3142/ debian bash
+#
+# Here, `dockerhost` is the IP address or FQDN of a host running the Docker daemon
+# which acts as an APT proxy server.
+FROM        ubuntu
```

Comment:
I think it's good practice to use specific tags there to make the Dockerfile persistent despite the upstream `latest` tag changes.